repo/packages/K/kubernetes/1/config.json

```diff
@@ -0,0 +1,195 @@
+{
+  "properties": {
+    "kubernetes": {
+      "description": "Kubernetes specific configuration properties",
+      "properties": {
+        "app-id": {
+          "default": "kubernetes",
+          "description": "Application ID.",
+          "type": "string",
+          "pattern": "^(?:[a-z][a-z0-9]*?(?:-[a-z0-9]+)*)$"
+        },
+        "docker-image": {
+          "default": "mesosphere/kubernetes:v1.0.1-v0.6.0-alpha",
+          "description": "Docker image used to launch the Kubernetes framework on DCOS.",
+          "type": "string"
+        },
+        "dns-suffix": {
+          "default": ".marathon.mesos",
+          "description": "This value is appended to the app-id value to form the canonical DNS name for the Kubernetes components.",
+          "type": "string",
+          "pattern": "^(?:\\.[a-z][a-z0-9]*?(?:-[a-z0-9]+)*)+$"
+        },
+        "enable-dns": {
```

Comment:
it looks like the shell scripting is expecting a bool now: `true` instead of `1`. so this needs to change